app/views/characters/_icon_view.haml

```diff
@@ -1,12 +1,15 @@
-- characters = name.characters.non_npcs.ordered unless local_assigns[:characters]
-- characters = characters.where(retired: false) unless local_assigns.fetch(:show_retired) { show_retired }
+-# locals: ( name: nil, characters: nil, show_template: false, show_user: false, hide_buttons: false, show_new_character_button: false, show_retired: show_retired() )
+-# show_template, show_user, and hide_buttons are to match list_section
```

Comment:
(does it need to match list_section? I think we pass in different parameters and it doesn't make sense to have those fields for the icon view given that we don't have those columns)

Reply:
We do a lot of passing things to partial_type and sometimes those things -- well, i ran into it with show_template but easier just to duplicate them all